Header header
float64 time    #ping time
float64 dt1     #time from trigger to center of water track cell
float64 dt2     #time from start of output message to center of water track cell
float64 batteryVoltage
float64 speedSound
float64 pressure
float64 temp